rust_dev_preview/examples/lambda/README.md

```diff
@@ -57,21 +70,26 @@ Other single action examples write directly to stdout, and can be executed with
 
 <!--custom.instructions.end-->
 
+
```

Comment:
This looks better, but it's missing the Scenario link section.

Reply:
Done